src/java.base/share/classes/java/util/ImmutableCollections.java

```diff
@@ -685,6 +687,7 @@ public int size() {
         }
 
         @Override
+        @ForceInline
```

Comment:
Good move here.  Another optimization bonus for List::of.

I suggest adding a comment for JDK maintainers:

`@ForceInline  //maximize chances of the JIT getting to the @Stable array`